src/obu.c

```diff
@@ -361,7 +369,13 @@ static avifBool parseAV2SequenceHeader(avifBits * bits, avifSequenceHeader * hea
     AVIF_CHECK(parseSequenceHeaderProfile(bits, header));
 
     // See av1_read_sequence_header() in avm.
-    AVIF_CHECK(parseSequenceHeaderFeatures(bits, header));
+    AVIF_CHECK(parseSequenceHeaderFrameMaxDimensions(bits, header));
+#if CONFIG_BLOCK_256
```

Comment:
The `CONFIG_BLOCK_256` and `CONFIG_LR_FLEX_SYNTAX` macros are not defined by libavif's cmake build system. Does this mean the code here is intended to be compiled by alternative build systems that define these macros?

Reply:
Just as for `CONFIG_LR_FLEX_SYNTAX`. It also helps keeping the parsing implementation in sync visually with libavm.